mmdet3d/datasets/waymo_dataset.py

```diff
@@ -26,524 +23,201 @@ class WaymoDataset(KittiDataset):
     Args:
         data_root (str): Path of dataset root.
         ann_file (str): Path of annotation file.
-        split (str): Split of input data.
-        pts_prefix (str, optional): Prefix of points files.
-            Defaults to 'velodyne'.
+        data_prefix (list[dict], optional): data prefix for point cloud and
```

Comment:
No need for optional for new version of mmdet3d
Still need to show the default value